float2words/float2words.py

```diff
@@ -55,22 +27,28 @@ def _split_float(number, precision):
     return whole_part, decimal_part
 
 
-def _convert_and_combine(number, lang, suffix):
-    """Convert number (integer) to words and combine with suffix.
+def float2words(number, lang, sfx1='', sfx2='', connector=', ', precision=2):
+    """Convert float to words by given optional parameters.
 
     Args:
-        number (integer): integer number.
+        number (float): float number.
         lang (str): language code (e.g. 'en_US', 'ru_RU', etc.).
-        suffix (str): suffix.
+        sfx1 (str): suffix for whole part of float number
+            (e.g. 'USD', '$', 'kg', etc.).
+        sfx2 (str): suffix for decimal part of float number.
+            (e.g. 'ct.', 'g', etc.)
+        connector (str): connector will be used between whole and
+            decimal parts expressed in words.
+        precision (int): number of digits after decimal point.
 
     Returns:
-        (str): combined number in words and suffix.
+        number_in_words (str): float number in words.
 
     """
-    # add whitespace before suffix if suffix is passed.
-    return ' '.join([
-        item for item in [
-            num2words(number, lang=lang),
-            suffix
-        ] if item
+    whole_part, decimal_part = _split_float(number, precision)
+    number_in_words = ''.join([
```

Comment:
Looks, good, just missing comment explaining why we are doing this, the way we are doing:)